From 678276525a48e265caa4b373c9b2b1b83d514d5e Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 13 Nov 2014 18:00:04 +0000
Subject: [PATCH 0614/1077] MANTIS 0034590 bugfix: mcontrol - Crash

mControl.exe
V1.0.2.17

- do no stop, when configuration of hosts file is not reachable



git-svn-id: svn://localhost/SelfServiceCommon/trunk@769 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Controller/inc/mcontrol_def.h      | 15 ++++++++++-----
 .../Massai/cpp/Controller/src/main.cpp            |  3 +--
 2 files changed, 11 insertions(+), 7 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
index 3fd1a84b..78ebecc8 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
@@ -9,8 +9,8 @@
  *
  ************************************************************************/
 
-#define FILE_VER_BIN     1,0,2,16
-#define FILE_VER_STR     "FileVersion", "1.0.2.16\0"
+#define FILE_VER_BIN     1,0,2,17
+#define FILE_VER_STR     "FileVersion", "1.0.2.17"
 
 /*!
 \file
@@ -18,19 +18,24 @@
 
 \page mControl.exe Release Notes - mControl.exe - platform startup, starts mudpate.exe, mstarter.exe; servant for systemControl interface
 
-@version 1.0.2.16
+@version 1.0.2.17
 
 \section change1 Change history
 
 mControl.exe
 
+\date 13.11.2014
+\li V1.0.2.17
+\li bugfix - SteT Mantis 0034590: mcontrol - Crash
+- do no stop, when configuration of hosts file is not reachable
+
 \date 28.03.2014 \li V1.0.2.16
                  \li modification - GiT Mantis 	0031431: mControl does not return to MAT_CLA in user shell mode
-                 
+
 \date 13.03.2014 \li V1.0.2.15
                  \li modification - GiT Mantis 0031488: PlatformSettings: support platform program path and platform data path to platform components
 
-                 
+
 \date 10.01.2014 \li V1.0.2.14
                  \li modification - GiT Mantis 0030362: MControl Change working dir to user local path
 
diff --git a/SelfServiceCommon/Massai/cpp/Controller/src/main.cpp b/SelfServiceCommon/Massai/cpp/Controller/src/main.cpp
index 8599ffdb..4f59f434 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/src/main.cpp
+++ b/SelfServiceCommon/Massai/cpp/Controller/src/main.cpp
@@ -626,8 +626,7 @@ int mDaemon_main( int argc, char const * const * argv )
       if(loadHostsFile() == -99)
       {
           //Configuration not started
-          TRACE(HERE,"ERROR: Configuration not started");
-          return -1;
+          TRACE(HERE,"ERROR: host file cannot be loaded; configuration not found");
       }
 
       // add scheduler tasks
-- 
2.41.0.windows.1

